src/graphql/data/util.tsx

```diff
@@ -147,8 +146,36 @@ export function isSupportedGQLChain(chain: Chain): chain is InterfaceGqlChain {
   return (UX_SUPPORTED_GQL_CHAINS as ReadonlyArray<Chain>).includes(chain)
 }
 
-export const BACKEND_CHAIN_NAMES = [Chain.Ethereum, Chain.Polygon, Chain.Optimism, Chain.Arbitrum, Chain.Celo] as const
-export const BACKEND_UNSUPPORTED_CHAIN_IDS = [ChainId.BNB] as const
+export function supportedChainIdFromGQLChain(chain: InterfaceGqlChain): ChainId
+export function supportedChainIdFromGQLChain(chain: Chain): ChainId | undefined
+export function supportedChainIdFromGQLChain(chain: Chain): ChainId | undefined {
+  return isSupportedGQLChain(chain) ? CHAIN_NAME_TO_CHAIN_ID[chain] : undefined
+}
+
+export function logSentryErrorForUnsupportedChain({
```

Comment:
Is this unused?

Reply:
nope it's used, it's a helper fn for Sentry errors relating to BE GQL providing data from an unsupported chain
<img width="287" alt="Screenshot 2023-07-06 at 11 36 58 AM" src="https://github.com/Uniswap/interface/assets/11512321/69d74f66-3b0b-4f55-a80d-2249d1444ddc">